app/models/user.rb

```diff
@@ -16,18 +16,16 @@ class User < ActiveRecord::Base
     :unless => Rails.env.test?
 
   belongs_to :user_cluster
-  has_many :leader_positions
 
   has_many :activities,    foreign_key: 'actor_id'
-  has_many :core_products, through: :core_team_memberships, source: :product
-  has_many :core_team_memberships, -> { where(is_core: true) }, class_name: 'TeamMembership'
-
-  has_many :assembly_assets
   has_many :awards, foreign_key: 'winner_id'
   has_many :choices
+  has_many :core_products, through: :core_team_memberships, source: :product
+  has_many :core_team_memberships, -> { where(is_core: true) }, class_name: 'TeamMembership'
```

Comment:
Line is too long. [92/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.